app/views/home/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if user_signed_in? %>
+
+<h3> You have successfully signed in </h3>
+<%= link_to "Create a new blog", {:action => 'new', :controller => 'blogs'}  %>
+<%= link_to "Manage your existing blogs",{:action => 'index', :controller => 'blogs'}  %>
```

Comment:
It would also be good to see some authentication in place for protected methods here. Devise provides a [method](https://github.com/plataformatec/devise#controller-filters-and-helpers) `authenticate_user!` which lets you restrict access to certain methods in your controller:

```ruby
class BlogsController < ApplicationController
  before_action :authenticate_user!, except: [:index, :show]
  # ...
end
```
will ensure that you're logged in before accessing any methods (except `index` or `show`).

You'll need to set up the `user_id` on blogs for that to work, then you can use things like:

```ruby
def set_blog
  @blog = current_user.blogs.find(params[:id])
end
```
which will ensure that you're only searching through the currently logged in user's blogs.

Hopefully that will give you enough to get the basic of authentication in place.

Reply:
That is cool -  I will give that a try.